handbook.md

```diff
@@ -0,0 +1,256 @@
+---
+layout: contribute
+title: Handbook
+---
+
+# How we make things
+
+## Branching and SCM
+
+We use git for source control. We follow this model in what we call the core projects ([Foreman](https://github.com/theforeman/foreman), [Smart Proxy](https://github.com/theforeman/smart-proxy), [Foreman installer](https://github.com/theforeman/foreman-installer) and [Foreman SELinux](https://github.com/foreman-selinux)) for branching:
```

Comment:
I know too many lists can start to hurt the eyes, but turning this into a list might make digestion easier.